libs/beacon/src/beacon_service.cpp

```diff
@@ -344,14 +344,6 @@ BeaconService::State BeaconService::OnCompleteState()
   block_entropy_being_created_->group_signature =
       active_exe_unit_->manager.GroupSignature().getStr();
 
-  // Check when in debug mode that the block entropy signing has gone correctly
-  if (!dkg::BeaconManager::Verify(block_entropy_being_created_->group_public_key,
-                                  block_entropy_previous_->EntropyAsSHA256(),
-                                  block_entropy_being_created_->group_signature))
-  {
-    FETCH_LOG_WARN(LOGGING_NAME, "Failed to verify freshly signed entropy!");
-  }
-
   // Save it for querying
```

Comment:
This test fails on all blocks except for the block at the start of the aeon as only in this block is the public key actually set correctly. Also the group signature is already verified correctly in the state previous to using manager.Verify()